djstripe/mixins.py

```diff
@@ -38,11 +38,20 @@ class PaymentsContextMixin(object):
 
     def get_context_data(self, **kwargs):
         context = super(PaymentsContextMixin, self).get_context_data(**kwargs)
+        selected_tag = context.get('selected_tag', False)
+        if selected_tag:
+            plan_list = [x for x in djstripe_settings.PLAN_LIST if x['tag'] == selected_tag]
```

Comment:
Should tags be held in the `Plan` model? Grabbing them from the settings seems unsustainable. 

CC: @kavdev